### PR TITLE
BTT Octopus DIN vertical mount

### PR DESCRIPTION
**Pull request by rsheng118**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
this adds a vertical bracket for new BTT octopus board

#### How was it tested? 
this is printed and mounted to voron 2.4

#### Screenshots (if appropriate)
inside the foler

#### Further notes
hope this will help some one

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -196,6 +196,7 @@ like so:
 | |[60mm Fan Cover](./Rhastlyn/60mmFanCover)|A easy to install cover for spec 60mm fans originally designed for a 2.4 build| :x: |:grey_question: |:heavy_check_mark: |:x:|
 | roboticator24 | [Touchscreen for OctoDash](./roboticator24/4inch_touchscreen_mount_for_v2.4) | Mount a 4-inch LCD touchscreen to your V2.4 in place of the usual 12864 panel to allow you to run the amazing OctoDash touch UI | :x: | :x: | :heavy_check_mark: |:x:|
 | rngSplat | [Printable Switchwire Deck Plate](./rngSplat/SW_Deck_Plate) | Print a deck plate for Switchwire with debossed Voron logo | :x: | :x: | :x: | :heavy_check_mark: |
+| rsheng118 | [BTT octopus DIN vertical mount](./rsheng118/BTT octopus DIN vertical mount) | DIN vertical mount for Big Tech Tree Octopus board | :x: | :x: | :heavy_check_mark: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:199]: Broken link, replace spaces with underscores, it should be:

    [BTT octopus DIN vertical mount](./rsheng118/BTT_octopus_DIN_vertical_mount)

File: printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# BTT octopus vertical mount
```
Comment (KiloQubit) [inline printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md:1]: Capitalize `Octopus`.

File: printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Overview
+This is a mount for new BTT octopus mount for rail. it mounts the board vertically to take full advantage of the fan to cool those stepper motor
```
Comment (KiloQubit) [inline printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md:4]: Capitalize `Octopus`.

Replace `for rail` with `for DIN rails`

Capitalize `It` at the start of the sentence.

Replace `stepper motor` with `stepper motor drivers`.

File: printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+remember to use the spacer.
```
Comment (KiloQubit) [inline printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md:6]: Capitalize `Remember` at the start of the sentence.

`spacer` should probably be plural `spacers` if you really need 4 of them.

You should mention here where the spacer should be used.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Your DIN_mount STLs are not in printable orientation.

Where does the spacer go?  between the board and the mounts at each screw hole, so you need 4 of them?  If so, please add `_x4` to that STL filename.

File: printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md (hunk 3 above)
Comment (KiloQubit) [reply printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md]: Still need to capitalize Octopus in this sentence, and replace `rail` with `rails`.

File: printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md (hunk 4 above)
Comment (KiloQubit) [reply printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md]: Please use plural `spacers`, since there are 4 of them.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates!

Still a few more README fixes remain.

File: printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 # BTT Octopus vertical mount
 
 ## Overview
-This is a mount for new BTT octopus mount for DIN rail. It mounts the board vertically to take full advantage of the fan to cool those stepper motor drivers.
+This is a mount for new BTT Octopus mount for DIN rails. It mounts the board vertically to take full advantage of the fan to cool those stepper motor drivers.
 
-Remember to use the spacer and place it in between the board and mounts.
+Remember to use spacers and place it in between the board and mounts.
```
Comment (KiloQubit) [inline printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md:6]: Replace `it` with `them`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Sorry, one more fix I missed before.

File: printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md (hunk 5 above)
Comment (rsheng118) [reply printer_mods/rsheng118/BTT_octopus_DIN_vertical_mount/readme.md]: nice catch, thanks. English isn't my strong suit, lol.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**